coreos-base/oem-gce/files/bin/enable-oslogin.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/bash
+
+# This is a stripped down version of the google oslogin control script
+
+set -e
+
+# nsswitch.conf changes
+grep oslogin /etc/nsswitch.conf || sed -i '/^passwd:/ s/$/ cache_oslogin oslogin/' /etc/nsswitch.conf
+
+# pam.d entry
+cat /usr/lib/pam.d/sshd /usr/share/oem/files/pam-sshd-config > /tmp/pam_sshd_config
+mv /tmp/pam_sshd_config /etc/pam.d/sshd
+#cp /usr/share/oem/files/pam-sshd-config /etc/pam.d/sshd
+
+# sshd_config changes
+cat /usr/share/ssh/sshd_config /usr/share/oem/files/sshd-config > /tmp/sshd_config
+mv /tmp/sshd_config /etc/ssh/sshd_config
```

Comment:
This script seems like it will overwrite files in `/etc` that the user might have configured with Ignition.  It should probably check if it's replacing anything other than an expected symlink.